x-pack/plugin/apm-data/src/yamlRestTest/resources/rest-api-spec/test/20_traces_ingest.yml

```diff
@@ -80,7 +80,20 @@ setup:
           - '{"@timestamp": "2017-06-22", "event": {"outcome": "unknown"}}'
 
           - create: {}
-          - '{"@timestamp": "2017-06-22", "event": {"outcome": "success"}}'
+          - '{
```

Comment:
I'm not sure you can split these over multiple lines (note: I'm also not sure you **can't**...).